sdk/servicebus/Microsoft.Azure.ServiceBus/tests/TokenProviderTests.cs

```diff
@@ -61,13 +33,98 @@ public async Task SasTokenWithLargeExpiryTimeShouldBeAccepted()
 
                 try
                 {
-                    var msg = await receiver.ReceiveAsync(TimeSpan.FromSeconds(5));
+                    var msg = await receiver.ReceiveAsync(TimeSpan.FromSeconds(5)).ConfigureAwait(false);
                 }
                 finally
                 {
-                    await receiver.CloseAsync();
+                    await receiver.CloseAsync().ConfigureAwait(false);
                 }
             });
         }
+
+        /// <summary>
+        /// This test is for manual only purpose.
+        /// </summary>
+        [Fact]
+        [LiveTest]
+        [DisplayTestMethodName]
+        public async Task QueueWithAadTokenProviderTest()
```

Comment:
>QueueWithAadTokenProviderTest [](start = 26, length = 29)

Why did you remove these tests?

Reply:
These are more like end-to-end integration tests that also rely on many other services, not sure if it's best practice to have it here.

---
In reply to: [289559465](https://github.com/Azure/azure-sdk-for-net/pull/6393#discussion_r289559465) [](ancestors = 289559465)

Reply:
@bainian12345 does sound like a good candidate for a separate project within the solution.

Reply:
I'm in agreement with Sean's suggestion.  There seem to be several "please don't run these as part of the automation" cases; a separate project or at least isolating them in a dedicated class would help to keep them from being confused with other tests.